framework/config/src/main/java/org/apache/cloudstack/framework/config/impl/ConfigDepotImpl.java

```diff
@@ -73,6 +76,7 @@
  */
 public class ConfigDepotImpl implements ConfigDepot, ConfigDepotAdmin {
     protected Logger logger = LogManager.getLogger(getClass());
+    protected final static long CONFIG_CACHE_EXPIRE_SECONDS = 30;
```

Comment:
We don't want this to be configurable - through some properties file or something?

Reply:
This could be a future improvement based on distributed framework or maybe a non-dynamic config. Currently, using something like db.properties key can create discrepancy between management servers